### PR TITLE
Adding note for values of DisableEnterpriseAuthProxy #2232

### PR DESCRIPTION
**Pull request by j0rt3g4**
Added Important Note at the end of article #2232

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/deployment/upgrade/upgrade-readiness-data-sharing.md [hunk 1]
```diff
@@ -45,4 +45,10 @@ In order to enable this scenario, you need:
 - Set the reg key HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\DataCollection\DisableEnterpriseAuthProxy to 0. If the value does not exist, create a new DWORD, name it DisableEnterpriseAuthProxy and set the value to 0. The deployment script will check this is configured correctly.
 - Set ClientProxy=User in bat.
 
+>[!IMPORTANT]
+> Using **Logged-in user's internet connection** with **DisableEnterpriseAuthProxy = 0** scenario is incompatible with ATP where the required value of that attribute is 1.(Read more here)[https://docs.microsoft.com/windows/security/threat-protection/windows-defender-atp/configure-proxy-internet-windows-defender-advanced-threat-protection]
```
Comment (nenonix) [inline windows/deployment/upgrade/upgrade-readiness-data-sharing.md]:
```suggestion
> Using the **Logged-in user's internet connection** with **DisableEnterpriseAuthProxy = 0** scenario is incompatible with ATP, where the required value of that attribute is 1. Read more about  [machine proxy and Internet connectivity settings](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-atp/configure-proxy-internet-windows-defender-advanced-threat-protection).
```

Comment (JohanFreelancer9) [reply windows/deployment/upgrade/upgrade-readiness-data-sharing.md]: Agree.

**Comment by e0i** [comment]
@jaimeo Would you review and merge this PR based on issue #2232 ? Thanks.

File: windows/deployment/upgrade/upgrade-readiness-data-sharing.md (hunk 1 above)
Comment (illfated) [reply windows/deployment/upgrade/upgrade-readiness-data-sharing.md]: 👍 I concur as well.

@j0rt3g4 : Please note that when adding or modifying MarkDown code containing URL links, the readable text is surrounded by square brackets `[ ]`, whereas the URL itself is surrounded by parentheses `( )`.

**Comment by j0rt3g4** [comment]
Oh did I mex them?

**Comment by illfated** [comment]
They were switched around. Here on the Github pages, if you use Ctrl-K after marking the text you want to add the URL to, it will place the brackets correctly for you. Even so, depending on your local editor software on your computer, it might not be as easy, but I don't know what kind of editing software you use to edit your commits.

**Comment by j0rt3g4** [comment]
Oops sorry I use vs :)

**Comment by illfated** [comment]
Fair enough. Myself, I use Git Extensions to handle my commits, whereas Notepad++ is my favorite text editor for code. If you can't find any method to implement simple auto insertions for URL link brackets+parentheses in Visual Studio, maybe you would like to check out if Notepad++ has got some macro function to do it for you.

**Pull request merged**